src/components/organisms/triggers/formParts/fileAndFunction.tsx

```diff
@@ -21,6 +23,25 @@ export const TriggerSpecificFields = ({ filesNameList }: { filesNameList: Select
 	const watchedFunctionName = useWatch({ control, name: "entryFunction" });
 	const watchedEventType = useWatch({ control, name: "eventType" });
 	const watchedFilter = useWatch({ control, name: "filter" });
+	const [selectedOption, setSelectedOption] = useState<SelectOption | null>(null);
```

Comment:
Can we use the form to keep the selected value and not in a state?
For example like connectionType.

Reply:
changed